website/src/css/custom.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --ifm-color-primary: #138cd3;
+  --ifm-color-primary-dark: #117ebe;
+  --ifm-color-primary-darker: #1077b3;
+  --ifm-color-primary-darkest: #0c5a87;
+  --ifm-color-primary-light: #159ae8;
+  --ifm-color-primary-lighter: #1e9feb;
+  --ifm-color-primary-lightest: #3dacee;
+}
+
+@media only screen and (min-device-width: 360px) and (max-device-width: 736px) {
```

Comment:
I think we can remove the empty media queries, although they're probably stripped out in the production build.